app/views/puppetclasses/_class_parameters.html.erb

```diff
@@ -15,6 +15,9 @@
   <tr id="puppetclass_<%= puppetclass.id %>_params[<%= lookup_key.id %>]"
     class="fields <%= 'overridden' if overridden %>">
     <%= content_tag(:td, puppetclass.name, :rowspan => lookup_keys.size, :class => "ellipsis") if index == 0 %>
+    <td class="ellipsis param_name">
+      <%= lookup_key.type == "PuppetclassLookupKey" ? "Smart Parameters":"Smart Variables"%>
```

Comment:
Please extract the strings for translation (https://theforeman.org/handbook.html#Stringsandtranslations).
You can also check the key type using `lookup_key.is_a? PuppetclassLookupKey` (reducing one string allocation per key)